website/content/docs/configuration/rpc.mdx

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+page_title: rpc Block in Agent Configuration
+description: >-
+   Configure Nomad's RPC Sessions multiplex protocol (rpc) system behavior in the `rpc` block of a Nomad agent configuration. Modify RPC multiplex session configuration, change tcp backlog, and keepalive interval, open socket timeout's.
+---
+
+# rpc Block in Agent Configuration
+
+<Placement groups={['rpc']} />
+
+This page provides reference information for configuring Nomad's RPC Session system behavior in the `rpc` block of a Nomad agent configuration. Modify RPC multiplex session configuration, change tcp backlog, and keepalive interval, open socket timeout's.
+
+```hcl
+rpc {
+  accept_backlog           = 256
+  keep_alive_interval      = "30s"
+  connection_write_timeout = "10s"
+  max_stream_window_size   = 262144
+  stream_open_timeout      = "75s"
+  stream_close_timeout     = "5m"
+}
+```
+
+## `rpc` Parameters
+
+- `accept_backlog` `(int: 256)` - AcceptBacklog is used to limit how many streams may be waiting an accept.
```

Comment:
We're copying these descriptions right out of the library docstrings, but the audience for those docstrings are consumers of the yamux library, not Nomad cluster admininstrators. For example:

```suggestion
- `accept_backlog` `(int: 256)` - The accept backlog limits how many RPC streams (requests) can be waiting to be accepted by the RPC server.
```

Take a pass over these rest of these fields and apply the same approach.